impl.go

```diff
@@ -83,18 +83,31 @@ func findInterface(iface string, srcDir string) (path string, id string, err err
 	if err != nil {
 		panic(err)
 	}
-	if len(f.Imports) == 0 {
+
```

Comment:
Add a line or two to the doc comment for this function explaining the added functionality with an example?